### ISSUE TITLE
integrate with beancount wallet

### ISSUE THREAD

**Issue by puncsky** [iotexproject/web-native-staking-v2#327]
probably update https://github.com/iotexproject/web-native-staking-v2/blob/460952308e855fbab71431cd09cdebf58f1a473b/src/shared/common/get-antenna.ts#L32 to 

signer = window.antennaSigner || new WsSignerPlugin("wss://local.iotex.io:64102"); // antennaSigner can injected by Browser Extension

**Comment by guo** [comment]
yes. because this will override ioPay desktop, we need to add a selector before connecting
